src/main/java/me/zeroeightsix/kami/process/HighwayToolsProcess.kt

```diff
@@ -0,0 +1,52 @@
+package me.zeroeightsix.kami.process
+
+import baritone.api.BaritoneAPI
+import baritone.api.IBaritone
+import baritone.api.pathing.goals.GoalNear
+import baritone.api.process.IBaritoneProcess
+import baritone.api.process.PathingCommand
+import baritone.api.process.PathingCommandType
+import me.zeroeightsix.kami.module.ModuleManager
+import me.zeroeightsix.kami.module.modules.misc.HighwayTools
+import me.zeroeightsix.kami.util.math.CoordinateConverter.asString
+
+/**
+ * @author Avanatiker
+ * @since 26/08/20
+ */
+class HighwayToolsProcess : IBaritoneProcess {
```

Comment:
change this to be an object, see #1327 which will be merged in a few days. Leaving this as a reminder.